ceph-releases/luminous/ubuntu/16.04/daemon/common_functions.sh

```diff
@@ -302,6 +302,19 @@ function get_part_uuid {
   blkid -o value -s PARTUUID "${1}"
 }
 
+function is_dmcrypt () {
+  # As soon as we find partitions with TYPE=crypto_LUKS or TYPE=plain
+  # on ${OSD_DEVICE} we can  assume this device is part of dmcrypt scenario.
+
+  # To keep compatibility with existing code
+  if [ -n "${1}" ]; then
+    local OSD_DEVICE
+    OSD_DEVICE="${1}"
+  fi
+  blkid -t TYPE=crypto_LUKS "${OSD_DEVICE}"* -o value -s PARTUUID 1> /dev/null || \
+  blkid -t TYPE=plain "${OSD_DEVICE}"* -o value -s PARTUUID 1> /dev/null || true
```

Comment:
plain encryption doesn't advertise itself like LUKS, or at least, it hasn't in our testing

Reply:
@alfredodeza true, it seems detecting a plain type encryption on a device is more challenging than luks. Will try to figure this out but it might not be as nice as luks.
Any idea is welcome.

Reply:
I think we should go without this anyway. PLAIN is not supported on the rest of the codebase so having it here doesn't change much.

Reply:
If it's not worth to have plain support right now for this PR, I won't spend too much time on this since we really need this patch: The ceph-ansible CI has been broken because of purge scenario which is constantly failing for very too long time.

We can still think about this and add this support later if needed I guess.